### DISCUSSION TITLE
Freya: Is there a forecast for updating the template for PrimeFaces 14?

### DISCUSSION THREAD

**Discussion by andersonneto** [primefaces/community#2658] [PrimeFaces Templates]
Do we have a forecast for updating the templates to version 14 of Primefaces? Because many features do not work in previous versions.

![image](https://github.com/user-attachments/assets/eae227d6-918f-4f99-9e36-f522f6bd38d4)

**Comment by melloware** [comment]
@andersonneto i have not seen any timeline but upgrading to 14 is not too bad.  See: https://github.com/primefaces/primefaces/wiki/Migration-Guide-for-Templates

I have upgraded a few of my templates to PF14 and it didn't take that long.

**Reply by andersonneto** [reply]
> @andersonneto i have not seen any timeline but upgrading to 14 is not too bad. See: https://github.com/primefaces/primefaces/wiki/Migration-Guide-for-Templates
> 
> I have upgraded a few of my templates to PF14 and it didn't take that long.

The problem is that the components of the latest version of the template have errors in the code, for example Poseidon, see what the Java code looks like:
![image](https://github.com/user-attachments/assets/8cbaff4d-55df-4740-b55d-daadee104b11)

![image](https://github.com/user-attachments/assets/ae96c639-baf6-4f23-9f43-4dc45449a636)

![image](https://github.com/user-attachments/assets/d6a22062-e109-497b-a458-4f9a63e74392)

Poseidon 6.0.0, Primefaces 13

**Reply by melloware** [reply]
Yep you have to make some Java changes. If you send me a DM on discord I can help you I have already upgraded Poseidon and Freya.

**Reply by andersonneto** [reply]
I would really appreciate it. As I'm new here, I don't know what the discord channel is or what a "DM" is :). Thank you very much in advance. A big hug.

**Reply by melloware** [reply]
@andersonneto its the Discord channel: https://discord.gg/gzKFYnpmCY

**Comment by NicolaIsotta** [comment]
It would be nice if PrimeTek shared a timeline for template updates, since PrimeFaces 13 is no longer updated on Maven.
I mean, PrimeFaces 14 was first released nearly 5 months ago, customers are waiting...

**Comment by tandraschko** [comment]
Plese see: https://x.com/primefaces/status/1970401400568918449
